envergo/moulinette/tests/test_views_haie.py

```diff
@@ -250,3 +250,69 @@ def test_moulinette_post_form_error(client):
     }
     assert "data" in error_event.metadata
     assert error_event.metadata["data"] == data
+
+
+@pytest.mark.urls("config.urls_haie")
+@override_settings(
+    ENVERGO_HAIE_DOMAIN="testserver", ENVERGO_AMENAGEMENT_DOMAIN="otherserver"
+)
+def test_result_p_view_with_hedges_to_remove_outside_department(client):
```

Comment:
Je te propose une docstring ici

```suggestion
def test_result_p_view_with_hedges_to_remove_outside_department(client):
    """Test if page displays when hedges to remove are outside department"""

```

Reply:
Je le rajoute.
Ceci dit pour les tests, je les commente en général avec [Given-When-Then](https://en.wikipedia.org/wiki/Given-When-Then)